mantidproject/mantid
# Add missing transmission run inputs to reflectometry GUI

This PR contains:
- addition of new input widgets for two missing stitching properties for transmission runs (ScaleRHSWorkspace and Params)
- addition of an input column on the per-theta defaults table for transmission processing instructions
- tidying to remove some hard-coded column indices in the tables (there are still some occurances of hard-coded indices but it's beyond the scope of this PR to fix all of them - I've added a note to #26025 to tidy this up in future).
- tidying of the Experiment and Instrument Settings tabs to improve the layout and improve some label names

**To test:**

- Open the ISIS Reflectometry interface
- Set the instrument to INTER and in the table enter run number 13460 and angle 0.5
- On the Experiment tab, enter 1st trans run 13463 and 2nd trans run 13464
- Enter Transmission Spectra = 4
- Untick ScaleRHSWorkspace
- Enter Transmission stitch params = -0.02
- On the Runs tab, click Process
- Check the history of IvsQ_binned_13460. It should have the following properties set in the input for ReflectometryISISLoadAndProcess:
  - FirstTransmissionRunList=13463
  - SecondTransmissionRunList=13464
  - Params=-0.02
  - ScaleRHSWorkspace=0
  - TransmissionProcessingInstructions=4

Fixes #26063 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
